### PR TITLE
Change deprecated bin/console server:start command to symfony server:…

### PR DESCRIPTION
**Pull request by nieuwenhuisen**
Change deprecated `bin/console server:start` command to `symfony server:start` in getting started section. And I added a link to de symfony local server documentation.

### CODE REVIEW

File: core/getting-started.md [hunk 1]
```diff
@@ -228,7 +228,7 @@ If you want to serialize only a subset of your data, please refer to the [Serial
 **You're done!**
 
 You now have a fully featured API exposing your entities.
-Run the Symfony app (`bin/console server:run`) and browse the API entrypoint at `http://localhost:8000/api`.
+Run the Symfony app with the [Symfony Local Web Server](https://symfony.com/doc/current/setup/symfony_server.html) (`symfony server:start`) and browse the API entrypoint at `https://localhost:8000/api`.
```
Comment (GregoireHebert) [inline core/getting-started.md:231]: I agree on the https part, but not on the command part.
If you only download the project itself, you do not have the symfony command available. But in both situation, the `bin/console` will work.

Comment (nieuwenhuisen) [reply core/getting-started.md:231]: So far as I know the https part is only for the Symfony Local Web Server and not for the `bin/console server:run` command. Also the `bin/console server:run` command is deprecated and will be probably removed in the future. That's why I suggest the new Symfony binary. This is not installed by default, but is very easy to install with the given link.

Comment (GregoireHebert) [reply core/getting-started.md:231]: I was not aware of it's deprecation :) 
Thanks for that 👍

Comment (alanpoulain) [inline core/getting-started.md:231]:
```suggestion
Run the Symfony app with the [Symfony Local Web Server](https://symfony.com/doc/current/setup/symfony_server.html) (`symfony server:start`) and browse the API entrypoint at `http://localhost:8000/api`.
```

**Comment by alanpoulain** [comment]
Thank you @nieuwenhuisen!

**Pull request merged**